docs/source/crud/piccolo_crud.rst

```diff
@@ -291,6 +291,41 @@ You can also use this on GET requests when retrieving a single row, for example:
 
 -------------------------------------------------------------------------------
 
+Content-Range header
+--------------------
+
+In some applications it can be practical to get information about the
+total number of records without invoking a separate call to
+the `count` endpoint. Piccolo API will supply this information in the
+`Content-Range` response header if `add_range_headers` is set to `True`.
+You can use the `range_header_plural_name` parameter to configure the
+"plural name" used in the `Content-Range` response header.
+
+The contents of the `Content-Range` header might look something like this
+for the "Movie" table: `movie 0-9/100`
```

Comment:
It's a bit annoying, but can we use double backticks? RST renders single backticks as italics, and double backticks as code blocks, which look slightly nicer.

Reply:
fixed!